### ISSUE TITLE
Dosent work with current ruversion of ruby 2.4.1

### ISSUE THREAD

**Issue by harsgak** [phstc/putsreq#34]
Getting error: Your Ruby version is 2.4.1, but your Gemfile specified 2.3.3
Tried changing ruby version in Gemfile but that led to dependency inconsistencies. I am not familiar with Ruby/Rails so i am not sure if that is to be done. 
Any way this error can be solved?

**Comment by phstc** [comment]
Hi @harsgak 

It currently supports only Ruby 2.3.3. Easiest thing would be to downgrade your Ruby version. I use [chruby](https://github.com/postmodern/chruby) in development, but there are other solutions such as [rvm](https://rvm.io/) for running multiple Ruby versions.

**Comment by harsgak** [comment]
Hi @phstc ,
I tried setting up rvm but couldn't get it to install on my machine. I am currently building ruby/rails from source and the current v2.4.1 just worked.
I will try figuring it out using chruby  sometime, but since I've moved to using [httpbin](https://httpbin.org) as I am more comfortable with python , it might not be soon. I think I'll just wait out until [putsreq](https://github.com/phstc/putsreq) comes to 2.4.x.
Thanks for the pointer to chruby though.
